### PR TITLE
Functional Tests : Migration TS (Install)

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Functional Tests : Migration TS (Install)
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Relative to #30263 
| Related PRs       | N/A
| How to test?      | Running https://github.com/boubkerbribri/ga.tests.ui.pr/actions/runs/3751273202
| Possible impacts? | N/A

### CODE REVIEW

File: tests/UI/campaigns/sanity/01_installShop/01_installShop.ts [hunk 1]
```diff
@@ -10,14 +10,12 @@ import homePage from '@pages/FO/home';
 // Import test context
 import testContext from '@utils/testContext';
 
-require('module-alias/register');
-
 const baseContext = 'sanity_installShop_installShop';
```
Comment (Progi1984) [inline tests/UI/campaigns/sanity/01_installShop/01_installShop.ts]:
```suggestion
const baseContext: string = 'sanity_installShop_installShop';
```

File: tests/UI/campaigns/sanity/01_installShop/01_installShop.ts [hunk 2]
```diff
@@ -84,9 +82,10 @@ describe('Install Prestashop', async () => {
   it('should fill shop Information form and go to step \'Content Configuration\'', async function () {
     await testContext.addContextItem(this, 'testIdentifier', 'goToContentConfiguration', baseContext);
 
-    await installPage.elementVisible(page, installPage.fourthStepFinishedListItem, 500);
+
```
Comment (Progi1984) [inline tests/UI/campaigns/sanity/01_installShop/01_installShop.ts]:
```suggestion
```

File: tests/UI/pages/install/index.ts [hunk 3]
```diff
@@ -1,17 +1,114 @@
 // Import pages
 import CommonPage from '@pages/commonPage';
-
-require('module-alias/register');
+import type {Page} from 'playwright';
```
Comment (Progi1984) [inline tests/UI/pages/install/index.ts]:
```suggestion

import type {Page} from 'playwright';
```

File: tests/UI/pages/install/index.ts [hunk 4]
```diff
@@ -222,7 +343,7 @@ class Install extends CommonPage {
    * @param timeout {number} Time to wait for step to finish
    * @returns {Promise<boolean>}
    */
-  async isInstallationStepFinished(page, step, timeout = 30000) {
+  async isInstallationStepFinished(page: Page, step: string, timeout = 30000): Promise<boolean> {
```
Comment (Progi1984) [inline tests/UI/pages/install/index.ts]:
```suggestion
  async isInstallationStepFinished(page: Page, step: string, timeout: number = 30000): Promise<boolean> {
```

**Review by Progi1984 — COMMENTED** [review]
Some feedbacks, but it's a good job ;)

File: tests/UI/campaigns/sanity/01_installShop/01_installShop.ts (hunk 1 above)
Comment (boubkerbribri) [reply tests/UI/campaigns/sanity/01_installShop/01_installShop.ts]: done

File: tests/UI/campaigns/sanity/01_installShop/01_installShop.ts (hunk 2 above)
Comment (boubkerbribri) [reply tests/UI/campaigns/sanity/01_installShop/01_installShop.ts]: done

File: tests/UI/pages/install/index.ts (hunk 3 above)
Comment (boubkerbribri) [reply tests/UI/pages/install/index.ts]: done

File: tests/UI/pages/install/index.ts (hunk 4 above)
Comment (boubkerbribri) [reply tests/UI/pages/install/index.ts]: Done, but typescript don't recommend this -> https://typescript-eslint.io/rules/no-inferrable-types/
they all be deleted when adding "plugin:@typescript-eslint/recommended" to eslint

Comment (Progi1984) [reply tests/UI/pages/install/index.ts]: As you want, but it's less clear on the type :(

**Review by Progi1984 — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @boubkerbribri

**Pull request merged**